src/test/java/cook/ParserTest.java

```diff
@@ -0,0 +1,206 @@
+package cook;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.HashMap;
+
+import org.junit.jupiter.api.Test;
+
+public class ParserTest {
+    private HashMap<String, Integer> init() {
+        HashMap<String, Integer> validCommandArgs = new HashMap<>();
+
+        validCommandArgs.put("bye", 1);
+        validCommandArgs.put("list", 1);
+        validCommandArgs.put("mark", 2);
+        validCommandArgs.put("unmark", 2);
+        validCommandArgs.put("delete", 2);
+        validCommandArgs.put("todo", 2);
+        validCommandArgs.put("deadline", 4);
+        validCommandArgs.put("event", 6);
+
+        return validCommandArgs;
+    }
+
+    @Test
+    public void byeTest() {
```

Comment:
Perhaps the test method names could be improved to better follow the three part format in the coding standard?

Reply:
🫡